Specs/createGlobe.js

```diff
@@ -25,7 +25,6 @@ define([
                 return 0.0;
             },
             _surface : {},
-            tileLoadedEvent : new Event(),
             imageryLayersUpdatedEvent : new Event(),
```

Comment:
I couldn't really figure this out - was `tileLoadedEvent` just removed? If so, why?

Reply:
Can you elaborate a bit more on _"was extremely costly"_ ? Like just raising this event is costly or does it involve having to keep track of extra things? I think it's probably fine if there is such a high cost to it.

Reply:
It was mostly this code:
https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/GlobeSurfaceTileProvider.js#L518

That was creating a new function for every loaded tile every frame.

I didn't spent a lot of time trying to think of an efficient way to implement this because it didn't seem very useful to me.